Earthfile

```diff
@@ -3,7 +3,7 @@ VERSION --try --raw-output 0.8
 
 PROJECT crossplane/crossplane
 
-ARG --global GO_VERSION=1.23.4
+ARG --global GO_VERSION=1.23.7
```

Comment:
i added a manual commit to keep this go version in the earthfile in sync with what renovate is bumping in go.mod. See https://github.com/crossplane/crossplane/issues/6343 for context 🤓